test/fail_compilation/ice10938.d

```diff
@@ -1,7 +1,8 @@
 /*
 TEST_OUTPUT:
 ---
-fail_compilation/ice10938.d(12): Error: no property `opts` for type `ice10938.C`
+fail_compilation/ice10938.d(13): Error: no property `opts` for type `ice10938.C`
+fail_compilation/ice10938.d(13):        potentially malformed `opDispatch`. Use an explicit instansiation to get a better error message
```

Comment:
Typo

Instantiation, not Instansiation

Reply:
fixed